src/router/index.js

```diff
@@ -103,7 +108,30 @@ const routes = [
           {
             path: "",
             name: "connect",
-            component: ConnectWallet
+            component: ConnectWallet,
+            children: [
+              {
+                path: "electrum-android",
+                component: ElectrumAndroid,
+                meta: {
+                  wallet: "electrum-android"
+                }
+              },
+              {
+                path: "fully-noded",
+                component: FullyNoded,
+                meta: {
+                  wallet: "fully-noded"
+                }
+              },
+              {
+                path: "specter-desktop",
+                component: SpecterDesktop,
+                meta: {
+                  wallet: "specter-desktop"
+                }
```

Comment:
We can add all the wallets here from the form select options:

https://github.com/louneskmt/umbrel-dashboard/blob/292238e43d33eefd5eedb3e6337e24aa5b706ae6/src/views/ConnectWallet.vue#L35